wradlib/tests/test_georef.py

```diff
@@ -37,9 +35,9 @@
 
 
 # this ensures objects from previous tests are cleaned
-@pytest.fixture(autouse=True)
-def ensure_gc():
-    gc.collect()
+# @pytest.fixture(autouse=True)
```

Comment:
@egouden before I merge this, can you check, if this garbage collection fixture is still needed?